lec15-data-viz.Rmd

```diff
@@ -1,4 +1,341 @@
+
 ---
-title: "Data visualization in R"
+title: Data visualization in R
+author: Lindsay Coome
 ---
 
+## Lesson preamble
+
+> ### Learning Objectives
+>
+> - Making better plots for outreach and communication.
+> - Review how to produce scatter plots with trendlines using ggplot.
+> - Advanced faceting and segmenting data.
+> - Altering asthetic features in ggplot.
+> - Make bar and other graphs using ggplot.
+> - Making interactive and quick graphs.
+> - Saving graphs.
+> 
+> ### Lesson outline
+>
+> - How to visualize data (20 min)
+> - ggplot basics (10 min)
+> - Basics and faceting (10 min)
+> - Aesthetics and plot types (25 min)
+> - Interactive and quick graphing (10 min)
+> - Saving graphs (5 min)
+
+-----
+
+## What makes a good graph?
+> #### Challenge
+>
+> What makes a bad graph? What makes an ugly graph to you?
+>
```

Comment:
In addition, Wikipedia actually has [a great article on misleading graphs](@lcoome Wikipedia actually has a great article on misleading graphs, bringing up the), bringing up many of the most common errors in terms of how the the visual elements are used.

Reply:
I had planned to do that, but I don't feel it's necessary to have it linked in the lesson material per se.

Reply:
Keep in mind, this material will be going on the website. And many of the students use the website during and after the lesson. Just something to consider

Reply:
We try to have essentially everything in or linked via the lecture material. The students use the materials as a reference when they work on the assignments and their final project so it would be great if you could add a couple of examples from those links and include the references if they want to see more.

This is also important for future iterations of the course.